modules/gdnative/pluginscript/pluginscript_script.cpp

```diff
@@ -317,20 +330,15 @@ Error PluginScript::reload(bool p_keep_state) {
 			_methods_rpc_mode[pi.name] = MultiplayerAPI::RPCMode(int(var));
 		}
 	}
-	// Manifest's attributes must be explicitly freed
-	godot_string_name_destroy(&manifest.name);
-	godot_string_name_destroy(&manifest.base);
-	godot_dictionary_destroy(&manifest.member_lines);
-	godot_array_destroy(&manifest.methods);
-	godot_array_destroy(&manifest.signals);
-	godot_array_destroy(&manifest.properties);
 
 #ifdef TOOLS_ENABLED
 /*for (Set<PlaceHolderScriptInstance*>::Element *E=placeholders.front();E;E=E->next()) {
 
         _update_placeholder(E->get());
     }*/
 #endif
+
+	FREE_SCRIPT_MANIFEST(manifest);
```

Comment:
Please also `#undef FREE_SCRIPT_MANIFEST` after the last use.

Reply:
Done !